src/components/pages/Navbar/Navbar.js

```diff
@@ -21,7 +21,7 @@ const Navbar = ({ isAuthenticated, userProfile, getProfile }) => {
   const [profilePic] = useState('https://joeschmoe.io/api/v1/random');
   const [user, setUser] = useState({});
   const [modal, setModal] = useState(false);
-  const [toggleStatus, setToggleStatus] = useState(true);
+  const [toggleStatus, setToggleStatus] = useState();
```

Comment:
All other commits look great, Jon.  I could be wrong here but, I believe you need to pass an empty dictionary "{ }"  into useState( ).

Reply:
Thanks, Steven, and also for your help with the DS side! The best practice is to pass the type of data that you expect to be receiving, such as an empty array/list or dictionary/obj. I should probably just set this to false because it will expect a boolean, couldn't decide before :).